docs/root/intro/life_of_a_request.rst

```diff
@@ -434,7 +434,7 @@ When ``decodeHeaders()`` is invoked on the :ref:`router <arch_overview_http_rout
 route selection is finalized and a cluster is picked. The HCM selects a route from its
 ``RouteConfiguration`` at the start of HTTP filter chain execution. This is referred to as the
 *cached route*. Filters may modify headers and cause a new route to be selected, by asking HCM to
-clear the route cache and requesting HCM to reevaluate the route selection. When the router filter
+clear the route cache and requesting HCM to reevaluate the route selection. Filters may also directly set this cached route selection via a `setRoute` callback. When the router filter
```

Comment:
wrap lines at 100 characters

Reply:
Is there a linter / script somewhere I can run to easily format rst? I tried building docs per https://github.com/envoyproxy/envoy/blob/main/docs/README.md but that didn't do anything

Reply:
I use `tools/code_format/check_format.py fix` to format but I'm not sure if it covers the RST files

Reply:
Yeah `tools/code_format/check_format.py fix` didn't work for this RST file, it's ok though I was able to wrap the lines manually!